grassroots-frontend/src/routes/Search.tsx

```diff
@@ -19,14 +20,18 @@ const ROWS_PER_PAGE = 10;
 
 function Search(): JSX.Element {
   const form = useForm<ContactSearchInDTO>({
-    resolver: classValidatorResolver(ContactSearchInDTO),
+    resolver: transformingClassValidatorResolver(
+      ContactSearchInDTO,
+      {},
+      { mode: "sync" },
+    ),
     mode: "onChange",
   });
 
   const [rowsToSkip, setRowsToSkip] = useState<number>(0);
 
   const searchParams: PaginatedContactSearchInDTO = {
-    contact: form.watch(),
+    contact: plainToInstance(ContactSearchInDTO, form.watch()),
```

Comment:
This definitely shouldn't have worked, but TypeORM happily took a string instead of a number...